basyx.aasrepository/basyx.aasrepository-http/src/test/java/org/eclipse/digitaltwin/basyx/aasrepository/http/TestAasRepositoryHTTP.java

```diff
@@ -263,6 +270,17 @@ public void updateNonExistingAas() throws IOException {
 		
 		assertEquals(HttpStatus.NOT_FOUND.value(), updateResponse.getCode());
 	}
+	
+	@Test
+	public void updateAasWithIdMismatch() throws IOException {
+		createDummyAasOnServer();
+		
+		String expectedAasJSON = getUpdatedAasIdMismatchJSONString();
```

Comment:
expectedAasJSON -> this is not expected, just the json for updating

Reply:
Renamed it